RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.10
+- Fixed unit tests 
+- Added github action workflow
+
```

Comment:
None of these changes change the service API or behavior, so IMO a version bump and release notes aren't necessary. That said, I don't particularly care either way, so I leave it up to you.

Reply:
Yeah, I see your point. But Gavin needs a new version to register it on ci.

Reply:
Why does it need to be registered? There's no bugfixes or new features... not that it really matters to me either way like I said before

Reply:
I'm Gavin btw, maybe you meant dylan?

Reply:
Sorry, I mean Dylan.😅
Because we need to release all the repos on ci after fixing broken tests, this is a jira ticket:https://kbase-jira.atlassian.net/browse/PTV-1587